test/old/testdir/test_signs.vim

```diff
@@ -1670,34 +1674,33 @@ func Test_sign_lnum_adjust()
   " changes made by this function.
   let &g:undolevels=&g:undolevels
 
-  " Nvim: make sign adjustment when deleting lines match Vim
-  set signcolumn=yes:1
+  " Nvim: deleting a line removes the signs along with it.
 
-  " Delete the line with the sign
-  call deletebufline('', 4)
-  let l = sign_getplaced(bufnr(''))
-  call assert_equal(4, l[0].signs[0].lnum)
+  " " Delete the line with the sign
+  " call deletebufline('', 4)
+  " let l = sign_getplaced(bufnr(''))
+  " call assert_equal(4, l[0].signs[0].lnum)
```

Comment:
Can this test simply be updated (or removed) instead of commenting it?

Reply:
No.

Reply:
If I understand correctly. The commented out test here isn't even vim behaviour which moves the signs instead.

Reply:
This test tests vim behavior, which we matched by setting `'signcolumn'` to `"yes:1". Since we decided we no longer want conditional sign removal, we can no longer use this test. Adapting it seems unnecessary, as signs now just match extmark behavior which is well tested. Apparently commenting it out is preferred to removing?

Reply:
The other test could be adapted I guess.

Reply:
I think yes. Commenting out two lines and adding two lines is a smaller change than commenting out all assertions. Note that there cannot be spaces between `"\` though.

Reply:
Not sure what you mean, what I had in mind was asserting that replacing those signs doesn't change the order:
```diff
diff --git a/test/old/testdir/test_signs.vim b/test/old/testdir/test_signs.vim
index 507efcefe..1beace7d3 100644
--- a/test/old/testdir/test_signs.vim
+++ b/test/old/testdir/test_signs.vim
@@ -1501,54 +1501,33 @@ func Test_sign_priority()
   call sign_place(3, '', 'sign3', 'Xsign',
               \ {'lnum' : 4, 'priority' : 20})
   let s = sign_getplaced('Xsign', {'group' : '*'})
-  call assert_equal([
+  let se = [
               \ {'id' : 3, 'name' : 'sign3', 'lnum' : 4, 'group' : '',
               \ 'priority' : 20},
               \ {'id' : 2, 'name' : 'sign2', 'lnum' : 4, 'group' : '',
               \ 'priority' : 20},
               \ {'id' : 1, 'name' : 'sign1', 'lnum' : 4, 'group' : '',
-              \ 'priority' : 20}],
-              \ s[0].signs)
+              \ 'priority' : 20}]
+  call assert_equal(es, s[0].signs)
 
   " Nvim: signs are always sorted lnum->priority->sign_id->last_modified
   " Last modified does not take precedence over sign_id here.
 
   " Place the last sign again with the same priority
-  " call sign_place(1, '', 'sign1', 'Xsign',
-  "             \ {'lnum' : 4, 'priority' : 20})
-  " let s = sign_getplaced('Xsign', {'group' : '*'})
-  " call assert_equal([
-  "             \ {'id' : 1, 'name' : 'sign1', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20},
-  "             \ {'id' : 3, 'name' : 'sign3', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20},
-  "             \ {'id' : 2, 'name' : 'sign2', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20}],
-  "             \ s[0].signs)
-  " " Place the first sign again with the same priority
-  " call sign_place(1, '', 'sign1', 'Xsign',
-  "             \ {'lnum' : 4, 'priority' : 20})
-  " let s = sign_getplaced('Xsign', {'group' : '*'})
-  " call assert_equal([
-  "             \ {'id' : 1, 'name' : 'sign1', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20},
-  "             \ {'id' : 3, 'name' : 'sign3', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20},
-  "             \ {'id' : 2, 'name' : 'sign2', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20}],
-  "             \ s[0].signs)
-  " " Place the middle sign again with the same priority
-  " call sign_place(3, '', 'sign3', 'Xsign',
-  "             \ {'lnum' : 4, 'priority' : 20})
-  " let s = sign_getplaced('Xsign', {'group' : '*'})
-  " call assert_equal([
-  "             \ {'id' : 3, 'name' : 'sign3', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20},
-  "             \ {'id' : 1, 'name' : 'sign1', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20},
-  "             \ {'id' : 2, 'name' : 'sign2', 'lnum' : 4, 'group' : '',
-  "             \ 'priority' : 20}],
-  "             \ s[0].signs)
+  call sign_place(1, '', 'sign1', 'Xsign',
+              \ {'lnum' : 4, 'priority' : 20})
+  let s = sign_getplaced('Xsign', {'group' : '*'})
+  call assert_equal(se, s[0].signs)
+  " Place the first sign again with the same priority
+  call sign_place(1, '', 'sign1', 'Xsign',
+              \ {'lnum' : 4, 'priority' : 20})
+  let s = sign_getplaced('Xsign', {'group' : '*'})
+  call assert_equal(se, s[0].signs)
+  " Place the middle sign again with the same priority
+  call sign_place(3, '', 'sign3', 'Xsign',
+              \ {'lnum' : 4, 'priority' : 20})
+  let s = sign_getplaced('Xsign', {'group' : '*'})
+  call assert_equal(se, s[0].signs)
 
   call sign_unplace('*')
```